sdks/java/core/src/main/java/org/apache/beam/sdk/util/PCollectionViews.java

```diff
@@ -165,8 +165,8 @@ public T getDefaultValue() {
             throw new RuntimeException("Unexpected IOException: ", e);
           }
         }
+        return defaultValue;
```

Comment:
I think the prior code is also fine, as it is lazy init. But it makes no real difference so this is fine too.